okapi-core/src/main/java/org/folio/okapi/bean/RoutingEntry.java

```diff
@@ -195,11 +194,6 @@ public String getPhaseLevel() {
    * @param methods HTTP method name or "*" for all
    */
   public void setMethods(String[] methods) {
-    for (String s : methods) {
```

Comment:
what is the reason for stripping this out?

Reply:
Because it never fires.. In fact HttpMethod.valueOf("GYF)".name() == "GYF".

Reply:
I've fixed this so that exception is thrown for invalid method.